### PR TITLE
doc: Improve the ceph fs set max_mds command

### PR DESCRIPTION
**Pull request by baruza**
Fixes: http://tracker.ceph.com/issues/21007
Signed-off-by: Bara Ancincova <bara@redhat.com>

### CODE REVIEW

File: doc/cephfs/multimds.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ result of commands.
 
     # fsmap e5: 1/1/1 up {0=a=up:active}, 2 up:standby
 
-    ceph fs set max_mds 2
+    ceph fs set <cephf-name> max_mds 2
```
Comment (tchaikov) [inline doc/cephfs/multimds.rst]: might want to use <fs_name> to be consistent with https://github.com/ceph/ceph/blob/master/doc/man/8/ceph.rst#fs.

also, in the context of cephfs, it's a little bit redundant to put `cephfs-name` here.

File: doc/cephfs/multimds.rst [hunk 2]
```diff
@@ -74,7 +74,7 @@ having just a single active MDS:
 ::
     
     # fsmap e9: 2/2/2 up {0=a=up:active,1=c=up:active}, 1 up:standby
-    ceph fs set max_mds 1
+    ceph fs set <cephfs-name> max_mds 1
```
Comment (tchaikov) [inline doc/cephfs/multimds.rst]: ditto.

File: doc/cephfs/multimds.rst (hunk 1 above)
Comment (tchaikov) [inline doc/cephfs/multimds.rst]: [off topic] the ceph(8) needs to be updated with the newly added subcommands of "fs". like `set`, `flag set`, and `{add|rm}_data_pool`.

**Comment by baruza** [comment]
@tchaikov 

Hi Kefu, I hope changed that, hope that everything is ok now.

**Comment by tchaikov** [comment]
@baruza nit, could you squash these two commit into a single one?

**Comment by baruza** [comment]
@tchaikov Done :)

**Pull request merged**